packages/terra-search-field/tests/wdio/search-field-spec.js

```diff
@@ -0,0 +1,252 @@
+      expect(browser.setValue.bind(browser, 'input', 'Lorem')).to
+        .throw('Element is not currently interactable and may not be manipulated');
+    });
+
+    it('should not accept clicks', () => {
+      expect(browser.click.bind(browser, '[class*="button"]')).to.throw(Error);
+    });
+  });
+
+  describe('With Placeholder', () => {
+    before(() => browser.url('/#/raw/tests/terra-search-field/search-field/search-field-with-placeholder'));
+
+    Terra.should.matchScreenshot('placeholder');
+    Terra.should.beAccessible({ rules: ignoredA11y });
+
+    it('should enter a search term', () => {
+      browser.setValue('input', 'Lorem');
+      browser.execute(() => {
+        // Removes the blinking cursor to prevent screenshot mismatches.
+        document.querySelector('input').style.caretColor = 'transparent';
+      });
+    });
+
+    Terra.should.matchScreenshot('with text');
+  });
+
+  describe('With Default Value', () => {
+    before(() => browser.url('/#/raw/tests/terra-search-field/search-field/search-field-default-value'));
+
+    Terra.should.matchScreenshot('default value');
+    Terra.should.beAccessible({ rules: ignoredA11y });
+
+    it('should enter a new search term', () => {
+      browser.click('input');
+      for (let i = 0; i < 7; i += 1) {
+        browser.keys('Backspace');
+      }
+
+      browser.addValue('input', 'Lorem');
+      browser.execute(() => {
+        // Removes the blinking cursor to prevent screenshot mismatches.
+        document.querySelector('input').style.caretColor = 'transparent';
+      });
+    });
+
+    Terra.should.matchScreenshot('with overwritten text');
+  });
+
+  describe('With Value', () => {
+    before(() => browser.url('/#/raw/tests/terra-search-field/search-field/search-field-with-value'));
+
+    Terra.should.matchScreenshot('default value');
+    Terra.should.beAccessible({ rules: ignoredA11y });
+
+    it('should try to enter a new search term', () => {
+      browser.click('input');
+      for (let i = 0; i < 4; i += 1) {
+        browser.keys('Backspace');
+      }
+
+      browser.addValue('input', 'Lorem');
+      browser.execute(() => {
+        // Removes the blinking cursor to prevent screenshot mismatches.
+        document.querySelector('input').style.caretColor = 'transparent';
+      });
+    });
+
+    Terra.should.matchScreenshot('unchanged text');
+  });
+
+  describe('Minimum Length', () => {
+    before(() => browser.url('/#/raw/tests/terra-search-field/search-field/minimum-length-search-field'));
+
+    it('should enter a short search term', () => {
+      browser.setValue('input', 'Lore');
+      browser.execute(() => {
+        // Removes the blinking cursor to prevent screenshot mismatches.
+        document.querySelector('input').style.caretColor = 'transparent';
+      });
+    });
+
+    Terra.should.matchScreenshot('with too short text');
+
+    it('should enter a long enough search term', () => {
+      browser.addValue('input', 'm');
```

Comment:
nit:  can we use `setValue` with an input that is obviously long enough here to improve clarity on what is happening in this test?